examples/webgpu_postprocessing_ao.html

```diff
@@ -70,8 +70,9 @@
 				const scenePass = pass( scene, camera );
 				scenePass.setMRT( mrt( {
 					output: output,
-					normal: transformedNormalWorld.directionToColor()
+					normal: transformedNormalWorld
 				} ) );
+				postProcessing.outputColorTransform = false;
```

Comment:
Adding this just temporarily to better visually debug the shader.

Reply:
Maybe you could try get `positionView` from MRT too? like:

```js
scenePass.setMRT( mrt( {
	output: output,
	normal: transformedNormalWorld,
	view: positionView
} ) );

const scenePassView = scenePass.getTextureNode( 'view' );
```

Then we find ways to optimize them too...

Reply:
Use `cameraProjectionMatrixInverse` inside a `QuadMesh.render()` it will use the current camera `OrthographicCamera` and not the camera used in `pass()`.

Reply:
Indeed, using `positionView` and the correct camera fixes the image!

I want to understand why the inline computation of `positionView` fails. I think this is related to the different clip space of WebGPU but I'm still not sure.

I'll push the camera fix first and hopefully a separate fix for position view later.